examples/nest/tsodyks_depressing.sli

```diff
@@ -21,7 +21,7 @@
  */
 
  /* BeginDocumentation
-Name: test_tsodyks_depressing - sli script for overall test of iaf_neuron model
+Name: test_tsodyks_depressing - sli script for overall test of iaf_psc_exp model
```

Comment:
The example already uses `iaf_psc_exp`, so I figured someone had changed the model and forgotten to update the comment. I did change it to `iaf_psc_alpha` first, and got confused when I couldn't find any `ìaf_neuron`s in the actual example. I had the same reasoning with the example below, tsdoyks_facilitating.sli.